app/models/jobstatusdata.rb

```diff
@@ -118,6 +120,43 @@ def extended_data_slurm(info)
     self
   end
 
+  # Store additional data about the job. (PBSPro-specific)
+  #
+  # Parses the `native` info function for additional information about jobs on PBSPRO systems.
+  #
+  # @return [Jobstatusdata] self
+  def extended_data_pbspro(info)
+    return unless info.native
+    self.account = info.group_list || ''
+
+    attributes = []
+    attributes.push Attribute.new "Cluster", self.cluster_title
+    attributes.push Attribute.new "PBS Id", self.pbsid
+    attributes.push Attribute.new "Job Name", self.jobname
+    attributes.push Attribute.new "User", self.username
+    attributes.push Attribute.new "Account", self.account
+    attributes.push Attribute.new "Walltime", (info.native.fetch(:Resource_List, {})[:walltime].presence || "00:00:00")
```

Comment:
Hmm, this should probably be a separate issue. I'd rather not use a non-release branch for the dependency, and once this is updated to the latest ood_core I'll be able to update all of the other walltime references for the other adapters.